site/_posts/2023-03-09-react-example-application.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: blog-post
+title: Using the FusionAuth React SDK in a React application
+description: In this guide, we'll showcase how to use the FusionAuth React SDK to implement the login, logout, and registration flows in a React application.
+author: Colin Frick
+category: blog
+image: blogs/react-example-application/react-example-application.png
+tags: tutorial tutorial-react tutorial-typescript react typescript oauth javascript
+excerpt_separator: "<!--more-->"
+---
+
+The React SDK from FusionAuth enables developers to implement the login, logout, and registration flows in a React application with ease.
+
+<!--more-->
+
+To show the React SDK in action, we will be using the FusionDesk application. FusionDesk is a simple help desk example application that allows users to create tickets and view them. The application is built using React and Node.js. It uses a server backend built using Ts.Ed, Node.js, and Typescript. The FusionDesk application is available on GitHub.
+
+{% include _image.liquid src="/assets/img/blogs/react-example-application/fusiondesk-tickets.png" alt="FusionDesk showing the main list of tickets" class="img-fluid" figure=false %}
+
+### Prerequisites
+
+To follow along with this article, you will need to have the following installed:
+
+* Node.js
+* Git
+* Docker with Docker Compose (optional)
+
+{% include _callout-tip.liquid
+content=
+"If you want to run the example application on another domain than `localhost`, you need to make sure that FusionAuth runs on the same domain as the backend of the application. This is a limitation of the Hosted OAuth Service Provider Endpoints."
+%}
+
+{% include _callout-important.liquid
+content=
+"This article was written using an early look at the FusionAuth Hosted OAuth Service Provider Endpoints. The feature may be not yet available in the latest version of FusionAuth."
```

Comment:
Point to the GH issue that is tracking that so someone can check it out. https://github.com/FusionAuth/fusionauth-issues/issues/1943